docs/api.md

```diff
@@ -76,7 +76,7 @@ function makeConsole() {
 }
 
 const newConsole = s.evaluate(`(${makeConsole})`, {consoleEndowment: console});
```

Comment:
Good catch! Do we want to do `const newConsole = s.evaluate(`(${makeConsole}())`, {consoleEndowment: console});` instead of calling `newConsole`?

Reply:
Good call, your way is a little more clear. I'll update the PR :+1: